tests/test_radiation.py

```diff
@@ -0,0 +1,150 @@
+"""Test module for abiotic.radiation.py."""
+
+from contextlib import nullcontext as does_not_raise
+
+import numpy as np
+import pytest
+
+from virtual_rainforest.models.abiotic import radiation
+
+
+@pytest.mark.parametrize(
+    argnames="elev, sun_frac, exp_tau, exp_err",
+    argvalues=[
+        pytest.param(
+            np.array([100, 1000]),
+            np.array([0.5, 1.0]),
+            np.array([0.501335, 0.770025]),
+            does_not_raise(),
+            id="standard_array_should_get",
+        ),
+        pytest.param(
+            np.array([100, 1000]),
+            np.array([-0.5, 1.5]),
+            (),
+            pytest.raises(ValueError),
+            id="value_error_sf",
+        ),
+    ],
+)
+def test_calculate_atmospheric_transmissivity(elev, sun_frac, exp_tau, exp_err):
+    """Test atmospheric tansmissivity across elevation and sunshine fractions."""
+
+    with exp_err as err:
+        result = radiation.calculate_atmospheric_transmissivity(elev, sun_frac)
+
+        # if no error is raised
+    if not err:
+        assert np.allclose(result, exp_tau)
+
```

Comment:
Failed: DID NOT RAISE <class 'virtual_rainforest.core.model.InitialisationError'>